guides/common/modules/proc_configuring-the-active-directory-authentication-source-on-projectserver.adoc

```diff
@@ -72,6 +73,16 @@ ifndef::orcharhino[]
 +
 For more information on GPOs, see {RHELDocsBaseURL}9/html/integrating_rhel_systems_directly_with_windows_active_directory/managing-direct-connections-to-ad_integrating-rhel-systems-directly-with-active-directory#how-sssd-interprets-gpo-access-control-rules_applying-group-policy-object-access-control-in-rhel[How SSSD interprets GPO access control rules] in _Integrating RHEL systems directly with Windows Active Directory (RHEL{nbsp}9)_.
 endif::[]
+.. If your {ProjectServer} is running in an IPV6-only network, set the `lookup_family_order` option in the `[domain/_ad.example.com_]` section of the `/etc/sssd/sssd.conf` file:
++
+[source, ini, options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[domain/_ad.example.com_]
+lookup_family_order = ipv6_only
+----
++
+When the DNS name of the AD server can be translated to both an IPv4 and IPv6 address but the IPv4 address is not accessible, SSSD requires `lookup_family_order` to translate the DNS name correctly.
```

Comment:
```suggestion
If the DNS name of the AD server can be translated to both an IPv4 and IPv6 address but the IPv4 address is not accessible, SSSD requires `lookup_family_order` to translate the DNS name correctly.
```

suggestion (non-blocking): Use "if" in favor of "when" because it's more about causality than a sequence. Your call. Same applies to line 48 in the other file.

Reply:
Applie, thanks!